gqlalchemy/models.py

```diff
@@ -113,7 +113,11 @@ def to_cypher(self) -> str:
         if self.batch_size is not None:
             query += f" BATCH_SIZE {self.batch_size}"
         if self.bootstrap_servers is not None:
-            query += f" BOOTSTRAP_SERVERS {self.bootstrap_servers}"
+            if isinstance(self.bootstrap_servers, str):
+                servers_field = f"'{self.bootstrap_servers}'"
+            else:
+                servers_field = str(self.bootstrap_servers)[1:-1]
+            query += f" BOOTSTRAP_SERVERS {servers_field}"
```

Comment:
Consider changing this if-else into one-liner

Reply:
can you help me understand how? Something like:
servers_field = f"'{self.bootstrap_servers}'" if isinstance(self.bootstrap_servers, str) else str(self.bootstrap_servers)[1:-1]
Seems too long